### ISSUE TITLE
Provide i18n for the graph right context menu

### ISSUE THREAD

**Issue by cigamit** [Cacti/cacti#1703]
Right now the right click context menu for the graph copy, etc. menu has no internationalization available.

**Comment by netniV** [comment]
Yeah I noticed this was hardcoded into the jquery.zoom.js file.  Since it's only required when that file is included, I think we could create an include function that would inject both the js and a localisation js with the text.

**Comment by cigamit** [comment]
All JavaScript i18n is handled in include/global_session.php.  Check it out.  Simple change.
